### PR TITLE
Removing unused arg PLATFORM

### PR DESCRIPTION
**Pull request by parambole**
Removing unused arg PLATFORM

### CODE REVIEW

**Review by gobbleturk — APPROVED** [review]
The PR title should say "unused" instead of used - can it also add a description (copy/paste of corrected titled is fine)

**Review by raymondzouu — COMMENTED** [review]
Thanks @parambole! Could you also make this change to the v5p configs?

File: MaxText/configs/v4/22b.sh [hunk 1]
```diff
@@ -20,7 +20,6 @@ echo "Running 22b.sh"
 # OUTPUT_PATH (Required, unless base_output_directory is already set in base.yml)
 # DATASET_PATH (Required, unless dataset_path is already set in base.yml)
 # RUN_NAME (Required, unless run_name is already set in base.yml or running with XPK/GKE)
-# PLATFORM (Optional, can be "gke" or "gce", default is "gce")
```
Comment (raymondzouu) [inline MaxText/configs/v4/22b.sh:23]: Could you also remove the `export PLATFORM="gce"` in line 34?

Comment (parambole) [reply MaxText/configs/v4/22b.sh:23]: Done

File: MaxText/configs/v5p/32b.sh [hunk 2]
```diff
@@ -6,10 +6,9 @@ echo "Running 32b.sh"
 # OUTPUT_PATH (Required, unless base_output_directory is already set in base.yml)
 # DATASET_PATH (Required, unless dataset_path is already set in base.yml)
 # RUN_NAME (Required, unless run_name is already set in base.yml or running with XPK/GKE)
-# PLATFORM (Optional, can be "gke" or "gce", default is "gce")
```
Comment (raymondzouu) [inline MaxText/configs/v5p/32b.sh:9]: Sorry one last thing, could you take out the `export PLATFORM="gce"` in this file on line 20?

File: MaxText/configs/v5p/64b.sh [hunk 3]
```diff
@@ -6,10 +6,9 @@ echo "Running 64b.sh"
 # OUTPUT_PATH (Required, unless base_output_directory is already set in base.yml)
 # DATASET_PATH (Required, unless dataset_path is already set in base.yml)
 # RUN_NAME (Required, unless run_name is already set in base.yml or running with XPK/GKE)
-# PLATFORM (Optional, can be "gke" or "gce", default is "gce")
```
Comment (raymondzouu) [inline MaxText/configs/v5p/64b.sh:9]: And in this file on line 20?

**Review by raymondzouu — COMMENTED** [review]
Sorry, just 2 last export PLATFORM remaining

**Review by raymondzouu — APPROVED** [review]
LGTM, thank you for cleaning this up!

**Pull request merged**